clients/gasManagement/gasStation.go

```diff
@@ -102,14 +117,21 @@ func (gs *gasStation) processLoop(ctx context.Context) {
 
 	for {
 		requestContext, cancel := context.WithTimeout(ctx, gs.requestTime)
-
+		nextRequestPoolingInterval := gs.requestPollingInterval
 		err := gs.doRequest(requestContext)
 		if err != nil {
-			gs.log.Error("gasHandler.processLoop", "error", err.Error())
+			gs.fetchRetries++
+			if gs.fetchRetries > gs.maximumFetchRetries {
+				gs.log.Error("gasHandler.processLoop", "error", err.Error())
+				gs.fetchRetries = 0
+			} else {
+				gs.log.Debug("gasHandler.processLoop", "message", err.Error())
+				nextRequestPoolingInterval = gs.requestRetryDelay
```

Comment:
should be this moved to L130 in order to change the request interval if the request errored either way?

Reply:
as discuss we can keep it like this in order to prevent spamming if api is down